web/app/css/styles.css

```diff
@@ -2,18 +2,17 @@
 
 :root {
   --white: #fff;
-  --buttonblue: #4076c4;
-  --messageblue: #2F80ED;
+  --indigoblue: #4076c4;
+  --royalblue: #2F80ED;
```

Comment:
there are some more references to `messageblue` in this file  as well as `service-mesh.css` that should be updated to this new color